ui/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8.5-buster
+
+WORKDIR /usr/src/app
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+COPY . /usr/src/app
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+
+CMD flask run --host=0.0.0.0 --port=${UI_PORT}
```

Comment:
let's run both of them with gunicorn

Reply:
it's not recommended to run gunicorn without having a nginix in front of it, and most of the articles I read don't use gunicorn for development purposes, and it also doesn't forward code errors (see attached picture) which will make development more cumbersome, thus I don't think it's a good idea to use gunicorn for development.

![error1](https://i.imgur.com/dJWiDmI.png)